docs/templates/RELEASE_NOTES-N.NN

```diff
@@ -0,0 +1,107 @@
+= Wikibase <N.NN> =
+
+THIS IS NOT A RELEASE YET
+
+Wikibase <N.NN> is currently a development branch, and is not recommended
+for use in production.
+
+== Upgrading notes for <N.NN> ==
+Don't forget to always back up your database before upgrading!
+
+Some specific notes for Wikibase <N.NN> upgrades are below:
+
+* …
+
+=== Configuration changes for system administrators in <N.NN> ===
+
+…
+
+==== New configuration ====
+* …
+
+==== Changed configuration ====
+* …
+
+==== Removed configuration ====
+* …
+
+=== New user-facing features in <N.NN> ===
+* …
+
+=== New developer features in <N.NN> ===
+* …
+
+=== External library changes in <N.NN> ===
+* …
+
+==== New external libraries ====
+* …
+
+===== New development-only external libraries =====
+* …
+
+==== Changed external libraries ====
+* …
+
+===== Changed development-only external libraries =====
+* …
+
+==== Removed external libraries ====
+* …
+
+=== Bug fixes in <N.NN> ===
+* …
+
+=== Action API changes in <N.NN> ===
+* …
+
+=== Action API internal changes in <N.NN> ===
+* …
+
+=== Languages updated in <N.NN> ===
+Wikibase supports over 300 languages. Many localisations are updated regularly.
+Below only new and removed languages are listed, as well as changes to languages
+because of Phabricator reports.
+
+* …
+
+=== Breaking changes in <N.NN> ===
+* …
+
+=== Deprecations in <N.NN> ===
+* …
+
+=== Other changes in <N.NN> ===
+* …
+
+== Compatibility ==
+Wikibase <N.NN> requires PHP 7.2.22 or later, and the following PHP extensions:
+
+* ctype
+* dom
+* fileinfo
+* iconv
+* json
+* mbstring
+* xml
+
+MySQL/MariaDB is the recommended DBMS. PostgreSQL or SQLite can also be used,
+but support for them is somewhat less mature.
+
+The supported versions are:
+
+* MySQL 5.5.8 or later
```

Comment:
is this true for Wikibase as well?

Reply:
Good question… also I doubt Wikibase supports PostgreSQL. Maybe just omit this and leave it to the installation instruction?

Reply:
Wikibase is only guaranteed to work on the MySQL and SQLite. Which versions I don't know, probably the closest to "confidence" we could get are the versions used on our CI builds + the MariaDB version WMF uses in production.

Postgresql I need to explicitly ask to not mention - it is not tested whatsoever, and, as of now, we don't want to make any claims about it being supported.

Reply:
removed this section